google-ads-examples/src/main/java/com/google/ads/googleads/examples/extensions/RemoveEntireSitelinkCampaignExtensionSetting.java

```diff
@@ -0,0 +1,213 @@
+    }
+
+    GoogleAdsClient googleAdsClient = null;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      System.exit(1);
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      System.exit(1);
+    }
+
+    try {
+      new RemoveEntireSitelinkCampaignExtensionSetting()
+          .runExample(googleAdsClient, params.customerId, params.campaignId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      gae.printStackTrace();
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+      System.exit(1);
+    }
+  }
+
+  /** Runs the example. */
+  // [START RemoveEntireSitelinkCampaignExtensionSetting]
+  private void runExample(GoogleAdsClient googleAdsClient, long customerId, long campaignId) {
+    // Creates an operation to remove a campaign extension setting.
+    CampaignExtensionSettingOperation extensionSettingOperation =
+        CampaignExtensionSettingOperation.newBuilder()
+            .setRemove(
+                ResourceNames.campaignExtensionSetting(
+                    customerId, campaignId, ExtensionType.SITELINK))
+            .build();
+
+    // Retrieves all sitelink extension feed items for a customer and campaign.
+    List<String> extensionFeedItemResourceNames =
+        getSitelinkExtensionFeedItems(googleAdsClient, customerId, campaignId);
+
+    // Constructs operations to remove the extension feed item operations.
+    List<MutateOperation> extensionFeedItemOperations =
+        extensionFeedItemResourceNames.stream()
+            .map(
+                feedItem ->
+                    MutateOperation.newBuilder()
+                        .setExtensionFeedItemOperation(
+                            ExtensionFeedItemOperation.newBuilder().setRemove(feedItem))
+                        .build())
+            .collect(Collectors.toList());
+
+    // Creates a list of operations that contains both the campaign extension setting and the feed
+    // item operations.
+    List<MutateOperation> allOperations = new ArrayList();
+    allOperations.add(
+        MutateOperation.newBuilder()
+            .setCampaignExtensionSettingOperation(extensionSettingOperation)
```

Comment:
I personally find it a little confusing that we create the CampaignExtensionSettingOperation independently, then wrap it in a MutateOperation later. WDYT of just inlining the CampaignExtensionSettingOperation creation here?
The advantage I see to that approach is that it emphasizes that the operations we're actually sending to the API are `MutateOperations`.